### PR TITLE
chore: update to Angular 2 RC 3

### PR DESCRIPTION
**Pull request by Foxandxss**
With new router for the demo.

### CODE REVIEW

File: demo/src/main.ts [hunk 1]
```diff
@@ -15,6 +16,6 @@ if (process.env.ENV === 'build') {
 
 bootstrap(AppComponent, [
     // These are dependencies of our App
-    ...ROUTER_PROVIDERS
+    ...APP_ROUTER_PROVIDERS
```
Comment (pkozlowski-opensource) [inline demo/src/main.ts]: I don't think we need destructing here, providers will be flattened.

**Comment by pkozlowski-opensource** [comment]
LGTM with one minor comment that can be safely ignored.

File: demo/src/main.ts (hunk 1 above)
Comment (Foxandxss) [inline demo/src/main.ts]: Yeah, probably not, remnants.

**Comment by pkozlowski-opensource** [comment]
Great stuff @Foxandxss !

**Pull request merged**